docs/adr/0015-copy-source-to-files.md

```diff
@@ -0,0 +1,25 @@
+# *Copying files from the source to the files folder*
+
+## Considered Alternatives
+* *Compiling files before copying them*
+* *Just copying the sources as they are*
+
+## Decision Outcome
+* Chosen Alternative: *Just copying the sources as they are*
+* Since the decision about an external IDE in [0014-use-eclipse-orion-as-editor](0014-use-eclipse-orion-as-editor.md)
```

Comment:
Sentence too long. The line of argumentation is: For supporting compiled languages, it is relied on an external IDE (see [ADR-0014](...)). Thus, the only left support is for scripting languages. In that case, the source can be directly used as files ("binary") in an artifact template.

To offer the same UI for editing scripting and compiled languages, the source files are always displayed in the "Source" tab.